src/clib-install.c

```diff
@@ -148,6 +148,8 @@ executable(clib_package_t *pkg) {
   E_FORMAT(&file, "%s-%s.tar.gz", reponame, pkg->version);
   E_FORMAT(&tarball, "%s/%s", tmp, file);
   rc = http_get_file(url, tarball);
+  if (0 != rc) goto cleanup;
```

Comment:
It will be nice to have a debug message here, like:

```
ERROR: download failed: %s: no such tag.
```

Reply:
I'm still new to the code, but maybe there's a debug/warning/error function we can add for debugging.

Reply:
I think you could add a `logger_error` there to display the error. With that, and after resolving the conflict, this could be merged imo. 

@jwerle

Reply:
🙏

Reply:
This seems to be fixed in master, by https://github.com/clibs/clib/commit/c410efa347b6e93ab5a33ad11da3717844f85261  :/